content/en/tracing/guide/trace_queries_dataset.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How Trace Queries Source Data
```

Comment:
I changed the title to be a bit simpler. Let me know if this still gets at what we are going for with this page. 😃

Reply:
```suggestion
Trace Queries Source Data
```